### ISSUE TITLE
Fixes for the CV-NLP-RL Intro Colab

### ISSUE THREAD

**Issue by charlesfrye** [wandb/examples#79]
## General
- [x] Bring notebook into repo for versioning
- [x] Periods at the end of sentences
- [x] Check consistency of `'` and `"`.
- [x] Follow PEP8 consistently
- [x] Slightly expand text or convert short text to headers
- [x] Improve navigation experience -- hide sidebar and add links or clean sidebar
- [x] Americanise spelling
- [x] Comment lines at a consistent rate/level of detail
- [ ] Define the garbage collection code at the top and re-use?
- [ ] Consistently use `-qqq` and `clear_output()` and `%%capture`

## Header
- [x] Fix "Create an Account" link
- [x] Better title and intro, fitting purpose as introductory notebook
- [x] Fix import order

## CV
- [x] Shorten the iNat model training -- val/loss stops going down after 10 epochs, more like 3 minutes than 7
- [x] Remove promise of precise time -- Colab compute is variable
- [x] Better text around YOLOv5

## NLP
- [x] Shorten the Yahoo Answers model training -- loss stops going down after one epoch

## RL
- [ ] Root-cause and fix crash in training code -- and consider removing dependence on `keras-rl2`, which is [no longer maintained](https://github.com/wau/keras-rl2)
- [ ] test_env commented code -- can that be removed? if so, do we need all video-writing utilties?
- [ ] Don't need `gym[all]`
- [ ] Increase duration of training for cartpole example -- up to a minute or two

**Comment by charlesfrye** [comment]
closed by ce0287924f9e0742a31f654be58d5ee1456af575
